pha-admin/src/main/java/com/store/pharmacy/exception/ExceptionHandlingController.java

```diff
@@ -26,38 +27,91 @@
 	@Autowired
 	private MessageRepository messageRepository;
 
+	private static final String DEFAULT_LOCALE_CODE = "en";
+
 	@ExceptionHandler({ DataNotFoundException.class })
 	public ResponseEntity<Object> handleDataNotFoundException(DataNotFoundException ex) {
 		String localizedErrorMessage = resolveLocalizedErrorMessage(ex.getMessage(), ex.getArgs());
 		FieldErrorDTO fieldErrorDTO = new FieldErrorDTO(null, localizedErrorMessage);
 		return new ResponseEntity<Object>(fieldErrorDTO, HttpStatus.NOT_FOUND);
 	}
-	
+
 	@ExceptionHandler({ DuplicateDataException.class })
 	public ResponseEntity<Object> handleDuplicateDataException(DuplicateDataException ex) {
```

Comment:
Replace the type specification in this constructor call with the diamond operator ("<>"). => new ResponseEntity<>(..)